plugins/generator-1.14.4/forge-1.14.4/templates/json/tag.json.ftl

```diff
@@ -15,6 +15,13 @@
             "${generator.getResourceLocationForModElement(value)}"
               <#if value?has_next>,</#if>
           </#list>
+      <#elseif data.type == "Entities">
+          <#list data.entities as value>
+              <#assign entity = generator.map(value.getUnmappedValue(), "entities", 1)?lower_case!"null">

```

Comment:
This will not work with custom entities (especially with custom registry name), also may not work with all vanilla entities as there is no guarantee that EntityType Forge name mapping (mapping ID 1) will match the entity machine name.

You would need to add new mappings for all entites for their registry names and implement system to support multiple `_mcreator_map_template` templates to offer one for registry names, or implement this inside this template for tag, but amount of changes required for this makes this PR's 2020.5 probability almost (or definite) 0%, unfortunately.

The need for new mappings is the reason this is not implemented yet originally, as I did not have time to implement this.

The same applies to 1.15.2.

Reply:
Thanks for your information.
Isn't providing registry names for all entities a way too big change?

Reply:
Yep, that's why I said the probability of this getting into core is sadly dangerously low.

Reply:
I suggest waiting with big (possibly breaking changes) for forks as suggested before. Now it is time for minor changes, and even for this very late time, as we are approaching final snapshot, and after that, no new features, just testing and bugfixing to deliver a stable release.